kernel/common/file.rb

```diff
@@ -104,6 +104,28 @@ def self.atime(path)
   end
 
   ##
+  # Returns the change time for the named file (the
+  # time at which directory information about the
+  # file was changed, not the file itself).
+  #
+  #  File.ctime("testfile")   #=> Wed Apr 09 08:53:13 CDT 2003
+  def self.ctime(path)
+    Stat.new(path).ctime
+  end
+
+  def self.birthtime(path)
+    Stat.new(path).birthtime
+  end
```

Comment:
I think this method should have a documentation comment above it as well to be consistent with the other two.